README.md

```diff
@@ -11,6 +11,18 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
 * [python_speech_features](https://pypi.python.org/pypi/python_speech_features)
 * [python sox](https://pypi.python.org/pypi/sox)
 
+
+## Install
+
+Manually install [Git Large File Storage](https://git-lfs.github.com/), then open a terminal and run:
+```
+git clone https://github.com/mozilla/DeepSpeech
+cd DeepSpeech
+git lfs fetch
```

Comment:
This does not work for me on OS X, see below. When I clone, the large files are already downloaded.

```bash
kdaviss-MBP:DeepSpeech kdavis$ git lfs fetch

Usage: 
  git-lfs [flags]
  git-lfs [command]

Available Commands: 
  clean                     Implements the Git clean filter
  env                       Show the current environment
  init                      Initialize the default Git LFS configuration
  logs                      View error logs
  ls-files                  Show information about Git LFS files
  pointer                   Build and compare pointers between different Git LFS implementations
  push                      Push files to the Git LFS server
  smudge                    Implements the Git smudge filter
  status                    Show information about Git LFS objects that would be pushed
  track                     Manipulate .gitattributes
  untrack                   Remove an entry from .gitattributes
  update                    Update local Git LFS configuration
  version                   Show the version number
  pre-push                  Implements the Git pre-push hook
  help [command]            Help about any command

 Available Flags:
      --help=false: help for git-lfs

Use "git-lfs help [command]" for more information about that command.
Error: unknown command "fetch"
Run 'help' for usage.

git-lfs: invalid command [`fetch`]
Run 'git-lfs help' for usage
```